lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -62,6 +73,21 @@ <h3 class="sr-only">${_("Print or share your certificate:")}</h3>
                             <span class="action-label">${_("Post on Facebook")}</span>
                         </a>
                       %endif
+                      % if share_settings.get('CERTIFICATE_TWITTER', False):
+                      <% twitter_url = 'https://twitter.com/intent/tweet?text=' + twitter_share_text + '&url='+ share_url%>
+                        <a
```

Comment:
@mattdrayer giving you a lil markup love. Here's what an HTML button element's markup would look like:

```html
<button
  data-tooltip="${_('Share on Twitter')}"
  data-action="${twitter_url}"
  class="action action-share-twitter btn btn-overlay btn-small icon-only"
  aria-haspopup="true"
  aria-expanded="false"
  title="${_('Share on Twitter')}"
  onclick="popupWindow('${twitter_url}', 'tweetWindow', 640, 480); return false;">
    <span class="action-label">${_("Tweet this Accomplishment")}</span>
    <i class="icon fa fa-twitter" aria-hidden="true"></i>
</button>
```

**NOTE**: Buttons don't have ``href`` attributes so I moved the value you had there into a ``data-action`` attribute, you'll want to update your JS to nab that instead.